website/docs/cloud-docs/integrations/run-tasks/index.mdx

```diff
@@ -99,6 +99,12 @@ Aqua Security’s Trivy integrates with Terraform Cloud run tasks to allow teams
 
 To get started, [sign up](https://info.aquasec.com/terraform-trivy) for the Trivy Run Tasks for Terraform Cloud integration, and follow the instructions in the [Terraform Cloud Run Tasks](https://aquasecurity.github.io/terraform-runtasks/0.1.1/getting_started) user documentation.
 
+### BlinkOps
+
+Blink makes it easy to add approvals checks via interactive Slack notifications to your Terraform workflows, or create event-based automations triggered by Terraform Cloud to enforce compliance, security, or business requirements.
```

Comment:
```suggestion
Blink makes it easy to add approvals checks using interactive Slack notifications to your Terraform workflows. You can also create event-based automations triggered by Terraform Cloud to enforce compliance, security, or business requirements.
```

Suggest changing "via" to "using" to avoid Latin words.